.gitignore

```
@@ -123,6 +123,9 @@ Doc/api/*.rst
 #Ignore mypy cache directory
 .mypy_cache
 
-#Ignore virtual environment file
+#Ignore virtual environments
 .env
+.venv
+env/
 venv/
+ENV/
```

```suggestion
ENV/

.coverage
```

Hello :wave: 

I know that is not the purpose of the PR, but maybe we can enjoy adding the `.coverage` ignore (generated by [coverage](https://coverage.readthedocs.io))?  It will avoid doing a dedicated PR.

 If you are okay, maybe you can update the PR name & description as well

---

I would rather have the coverage covered in a separate PR that also adds text to the README.md how to get the test coverage of biopython.

---

Maybe not the README.md, but the CONTRIBUTING.rst

---

OK, changing my mind... we can make this a local development PR.